src/plugins/index_pattern_management/public/components/create_index_pattern_wizard/components/step_data_source/components/header/header.tsx

```diff
@@ -23,10 +23,12 @@ import {
   DataSourceRef,
   IndexPatternManagmentContext,
 } from 'src/plugins/index_pattern_management/public/types';
+import semver from 'semver';
 import { useOpenSearchDashboards } from '../../../../../../../../../plugins/opensearch_dashboards_react/public';
 import { getDataSources } from '../../../../../../components/utils';
 import { DataSourceTableItem, StepInfo } from '../../../../types';
 import { LoadingState } from '../../../loading_state';
+import * as pluginManifest from '../../../../../../../opensearch_dashboards.json';
```

Comment:
nit: can we use absolute path here instead of relative path with directory traversal?

Reply:
yes, this one just to keep the save relative style as existing imports lol